pkg/render/apiserver.go

```diff
@@ -2290,6 +2292,26 @@ func (c *apiServerComponent) multiTenantManagedClusterAccessClusterRoles() []cli
 	return objects
 }
 
+// managedClusterClusterRoles creates ClusterRoles for accessing the ManagedCluster API
+func (c *apiServerComponent) managedClusterClusterRoles() []client.Object {
```

Comment:
Does this need to return a slice any more? It's only a single cluster role at this point.